ebsco/edsapi-koha-plugin
# Update to correct linking issues with title links

This small PR corrects an issue where the plugin path was not getting set correctly in links to the detailed record for EDS items.

To Test:
http://koha-2105.ebsco-gss.net/
Do a search with the Discovery dropdown selected.
Click on a detailed record link.
Verify that the link does take you to the detailed record.
